ArduPlane/reverse_thrust.cpp

```diff
@@ -71,6 +71,7 @@ bool Plane::allow_reverse_thrust(void) const
         allow |= (g.use_reverse_thrust & USE_REVERSE_THRUST_RTL);
         break;
     case Mode::Number::CIRCLE:
+    case Mode::Number::TAKEOFF:
```

Comment:
why not use a new USE_REVERSE_THRUST_TAKEOFF?

Reply:
in what circumstances would you want reverse thrust for takeoff?

Reply:
I can't think of one, which is why [it was never allowed in the first place](https://github.com/ArduPilot/ardupilot/blob/master/ArduPlane/reverse_thrust.cpp#L37-L40) so you're the one introducing it. I would prefer to see it just return false on it's own switch line.

Reply:
good point, fixed.
also fixed an issue with the reverse thrust bit handling code